src/methods/users.js

```diff
@@ -11,12 +11,19 @@ export default function users(): Object {
       });
     },
 
-    photos: (username: string, page: number = 1, perPage: number = 10, orderBy: string = "latest") => {
+    photos: (
+      username: string,
+      page: number = 1,
+      perPage: number = 10,
+      orderBy: string = "latest",
+      stats: bool = false,
+    ) => {
```

Comment:
I was thinking that, as I much prefer named parameters! But it would break the convention we have across our module. We'd have to change it for every function, and publish a new major version.

Something to look into for the near future.